interface/.dockerignore

```diff
@@ -1,4 +1,3 @@
 bin
 build
-node_modules
-vendor
+adage/adage/config.py
```

Comment:
Yep, it's true, I added that one to the root-level .dockerignore file, so I'll remove this. I guess I was thinking of the use when somebody was trying to run the Docker container to build the interface from a folder other than `interface/` but I guess that wouldn't matter because the paths in this file are relative to where this file is.